lib/plugins/features/movies/controller.js

```diff
@@ -6,5 +6,41 @@ exports.create = (payload) => {
 
   return new Movie().save(payload)
   .then((movie) => new Movie({ id: movie.id }).fetch());
-  
+
 };
+
+exports.display = (params) => {
+
+  if (params === undefined || Object.keys(params).length === 0) {
+    return new Movie().fetchAll();
+  }
+
+  else if (params.release_year !== undefined) {
+    return new Movie().query(function(qb) {
+      qb.where('release_year', '=', params.release_year);
+    })
+    .fetchAll();
+  }
+
+  else if (params.start_year !== undefined && params.end_year !== undefined) {
+    return new Movie().query(function(qb) {
+      qb.where('release_year', '>=', params.start_year).andWhere('release_year', '<=', params.end_year);
+    })
+    .fetchAll();
+  }
+
+  else if (params.title !== undefined && params.fuzzy === 'no') {
+    return new Movie().query(function(qb) {
+      qb.where('title', '=', params.title);
+    })
+    .fetchAll();
+  }
+
+  else if (params.title !== undefined && params.fuzzy === 'yes') {
```

Comment:
One of my favorite parts of JS is how slick conditionals can be with undefined. 

1. `params.title !== undefined` returns boolean
2. `params.title` returns the title or `undefined` or `null`

The second is preferable in this case because it functions equivalently to 2 AND a `null` `params.title` would also not work.